### ISSUE TITLE
Elevated circulating adipic acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6022]
**Preferred term label:** Elevated circulating adipic acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of adipic acid in the blood is above the normal range., PMID: 5031780

**Parent term (use hpo.jax.org/app)** Dicarboxylic acidemia HP:0040145

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
see https://github.com/obophenotype/human-phenotype-ontology/issues/6021
